doc/input-output-reference/src/standard-output-reports/output-table-summaryreports.tex

```diff
@@ -45,7 +45,7 @@ \subsubsection{Predefined Annual Summary Reports}\label{predefined-annual-summar
 \item
   Skylight-Roof Ratio table for envelope which includes the roof area and the skylight area and the ratio of the two. This includes all surfaces with a tilt of less than 60 degrees.
 \item
-  Zone Summary includes internal load summary for each zone including area, if conditioned, volume, multipliers, gross wall area, window area, design lighting, design people, design plug and process.
+  Zone Summary includes internal load summary for each zone including area, if conditioned, volume, multipliers, above ground gross wall area, underground gross wall area, window area, design lighting, design people, design plug and process.
```

Comment:
What's the standard terminology, if there is any, in std 90 or elsewhere?  "Above Ground"/"Underground" or "Below Ground" or should they be "Above Grade" and "Below Grade" or?